File: python/ej8/PruebaTerminal.py
'''
Prueba de la clase terminal.

@author: Rafael Infante
'''
from programasPoo.Terminal import Terminal

t1=Terminal("663654985")
t2=Terminal("703644864")
t3=Terminal("713654464")

t1.llama(t2, 340)
t1.llama(t3,500)

print(t1)
print(t2)
print(t3)

File: java/ej8/PruebaTerminal.java
package programasPoo;

public class PruebaTerminal {

  public static void main(String[] args) {
    //instancias 
    Terminal t1=new Terminal("663654985");
    Terminal t2=new Terminal("678456598");
    Terminal t3=new Terminal("636658247");
    
    System.out.println(t1);
    System.out.println(t2);
    
    t1.llama(t2, 200);
    t2.llama(t3, 250);
    t1.llama(t3, 150);
    
    System.out.println(t1);
    System.out.println(t2);
    System.out.println(t3);
  
    
  }

}
